pkg/models/user.go

```diff
@@ -29,6 +29,8 @@ type User struct {
 	IsAdmin          bool `gorm:"default:false"`
 	IsCommunityAdmin bool `gorm:"default:false"`
 	InvitationCode   string
+	Reward           int  `gorm:"default:0"`
```

Comment:
No need to set default value here, it's already the default.

```suggestion
	Reward           int
```